### ISSUE TITLE
Beautifier Pipeline: Multiple/cascading beautifiers per language

### ISSUE THREAD

**Issue by Glavin001** [Unibeautify/unibeautify#4]
See the [original issue](https://github.com/Glavin001/atom-beautify/issues/457) for more details.
- [x] Options values will determine the default beautifiers chosen
  - This allows configuration files, such as `.jsbeautifyrc` to contain a field `default_beautifiers` and therefore be automatically configurable while switching different projects.
  - See https://github.com/Glavin001/atom-beautify/issues/1041
- [x] `default_beautifiers` will accept an array (of course) and process the text sequentially.

**Comment by eivindml** [comment]
Any progress on this? Been looking for a feature like this for quite some time. :)

**Comment by Glavin001** [comment]
No progress yet, however I just finished my final exams for University yesterday! So now I have a little more time to get us back on track 😃

**Comment by Glavin001** [comment]
Going to use option key `beautifiers` instead of `default_beautifiers`.

**Comment by Glavin001** [comment]
Published to v0.5.0!

**Comment by Glavin001** [comment]
Example `.unibeautifyrc.json` file:

```json
{
  "JavaScript": {
    "beautifiers": ["Pretty Diff", "JS-Beautify", "Prettier"]
  }
}
```
